deploy/examples/create-external-cluster-resources.py

```diff
@@ -463,6 +466,15 @@ def gen_arg_parser(cls, args_to_parse=None):
             required=False,
             help="comma-separated list of the k8s cluster failure domain values corresponding to each of the pools in the `topology-pools` list",
         )
+        output_group.add_argument(
+            "--cephx-key-rotate",
+            default="",
+            required=False,
+            help="Enable cephx key rotation for the users created by this script, this will create a new user with suffix `.{x}` and update the secrets with the new key,"
+            + "use for rotataing csi and healthchecker users keys"
+            + "For usage, we need to set `--cephx-key-rotation rotate` so key rotation can happen"
+            + "In case of reverting the keys to the one older key, we need to set `--cephx-key-rotation revert`",
```

Comment:
Docs and usage text should avoid "we" "us" "you" and other similar words. Also, since this is not a multiline string, make sure periods separate the sentences.

```suggestion
            help="Enable cephx key rotation for the users created by this script. This will create a new user with suffix `.{x}` and update the secrets with the new key."
            + "Set `--cephx-key-rotation rotate` to initiate rotation."
            + "To revert keys to the prior generation, set `--cephx-key-rotation revert`.",
```

Also, IMO, "Enable cephx key rotation for the users created by this script" already implies that CSI and health checker user keys are rotated. I don't think users should need reminded of what keys are created by the script, but if you think they do, I would suggest that for the first help line.